tests/scripts/all.sh

```diff
@@ -1322,8 +1339,8 @@ component_check_python_files () {
 }
 
 component_check_generate_test_code () {
-    msg "uint test: generate_test_code.py"
-    record_status ./tests/scripts/test_generate_test_code.py
+    msg "unit test: generate_test_code.py"
+    record_status ./tests/scripts/test_generate_test_code.py -q 2>/dev/null
```

Comment:
I don't like hiding stderr. I'd strongly prefer modifying `test_generate_test_code.py` to not print anything to stderr if it succeeds.

Reply:
I agree. I could not easily find a way to do that, though.

Reply:
I had a look at `unittest` and I didn't find a way to make it truly quiet either. But I really don't want to hide errors in non-quiet mode.

I think the best design here would be to have `2>/dev/null` only in `QUIET` mode, and remove `-q`.

Reply:
Ok, so what I did is redirect stderr to stdout unconditionally, which is then redirected to `/dev/null` only in quiet mode. (And I removed the `-q`.)